### ISSUE TITLE
NotificationService cannot be injected in ErrorHandler

### ISSUE THREAD

**Issue by gyoshev** [telerik/kendo-angular#1989]
## I'm submitting a...

* Bug report 

## Current behavior

Injecting the `NotificationService` class in an `ErrorHandler` class triggers the following error:

```
Provider parse errors:
Cannot instantiate cyclic dependency! ApplicationRef
```

## Expected behavior

The `NotificationService` should be injectable in an `ErrorHandler` class, because it is likely that the developer wants to notify the user of an error.

## Minimal reproduction of the problem with instructions

https://stackblitz.com/edit/angular-3yegcw

## Workaround

Until this issue is fixed, the `NotificationService` can dynamically be injected whenever the ErrorHandler needs to report the error. See [this example](https://stackblitz.com/edit/angular-ywdiic-zepdnj?file=app%2Fmy-error-handler.service.ts) for a working demo.

**Comment by elena-gancheva** [comment]
The fix is available in the latest `dev` version of the package.

**Comment by elena-gancheva** [comment]
Shipped with  `v1.2.0` of the package.
